dev/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd _backend
+composer up
+cd ../
+npm i -g npm
```

Comment:
I don't think you should do a global install here of node modules. You should be fine doing a plain `npm install`

Reply:
didn't really need that global install, it's already been removed